MiniApp.podspec

```diff
@@ -26,10 +26,15 @@ Pod::Spec.new do |s|
   s.ios.deployment_target = '11.0'
 
   s.source_files = 'MiniApp/Classes/**/*.swift'
-  s.resources = ['MiniApp/**/*.{xcassets,js,pdf,xib}','js-miniapp/bridge.js']
-  s.resource_bundle = {"Localization" => ["MiniApp/*.lproj/*.strings"]}
+  s.resource_bundle = {
+    "MiniApp" => ['MiniApp/**/*.{xcassets,js,pdf,xib}','js-miniapp/bridge.js'],
+    "Localization" => ["MiniApp/*.lproj/*.strings"]
+  }
+
+  s.static_framework = true
 
   s.dependency 'RSDKUtils', '>= 1.1.0'
   s.dependency 'ZIPFoundation'
+  s.dependency 'Google-Mobile-Ads-SDK'
```

Comment:
Can we make a subspec for this, like `MiniApp/Ads` so developers who don't want to depend from Google Ads can. What do you think @corycaywood ?

Reply:
Yeah, I agree. Google Ads SDK should be an optional dependency because not every Host App will want to use Admob.